docs/azure/sdk/snippets/protocol-convenience-methods/SCMConvenience/Program.cs

```diff
@@ -0,0 +1,14 @@
+﻿using OpenAI;
+using OpenAI.Chat;
+using System.ClientModel;
+
+// Create the client
+OpenAIClient client = new("your-openai-api-key");
+ChatClient chatClient = client.GetChatClient("gpt-4");
+
+// Call the convenience method
+ChatCompletion completion
+    = chatClient.CompleteChat("What is Azure?");
```

Comment:
Include "Microsoft" for consistency with the other code samples.
```suggestion
    = chatClient.CompleteChat("What is Microsoft Azure?");
```